server/src/routes/index.ts

```diff
@@ -0,0 +1,10 @@
+import { Router } from "express";
+import authRouter from "./auth";
+import userRouter from "./user";
+
+const userAndAuthRouter = Router();
```

Comment:
Change this name incase we have more than just user and auth. Make it applicable to any routes created.